### DISCUSSION TITLE
Does the Cypress direct download provide the full command line functionality?

### DISCUSSION THREAD

**Discussion by KvnSearle** [cypress-io/cypress#17082] [Questions and Help] [answered]
It seems whatever command I use all I can do is fire up the test runner.

For example, with the Cypress.exe installed at D:\Applications\Cypress, each of the following result in an empty test runner popping up
	D:\Applications\Cypress\Cypress.exe info
	D:\Applications\Cypress\Cypress.exe run -spec cypress\integration\demo.spec.js
	D:\Applications\Cypress\Cypress.exe run -spec "cypress\integration\demo.spec.js"

The only variation I can get is using the project operator as follows
	D:\Applications\Cypress\Cypress.exe run --project D:\demo\cypress-tests-main-laptop\cypress-tests-main
which sees the test runner pop up with the tests listed but given the run command is used I would expect them to execute.

I can't work out what is going on.

Any pointers will be gratefully received.

**Comment by jennifer-shehane** [comment, marked as answer]
@KvnSearle No, you can't run the CLI commands with the direct download option of Cypress. It's provided as a way to quickly try out the product. If you have a project setup and are ready to run Cypress in the terminal, you'll need to install Cypress as a dependency via yarn or npm. https://on.cypress.io/installing-cypress#Installing

**Reply by KvnSearle** [reply]
Thanks for getting back to me.
The only limitation of the direct download I can see documented at this time is the fact that recording runs to the Dashboard is not possible
Would be possible to add the fact that you can't run the CLI commands either and any other limitation for that matter?
Once again thanks for the response.
